### PR TITLE
impr: Refactor integration tests to use AbstractController

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java [hunk 1]
```diff
@@ -70,4 +95,21 @@ private AboutToStartOrSubmitCallbackResponse postEvent(String path, CaseDetails
...
         }
     }
+
+    private AboutToStartOrSubmitCallbackResponse postEvent(String path, byte[] data, int expectedStatus) {
+        try {
+            MvcResult response = mockMvc
+                .perform(post(path)
+                    .header("authorization", userAuthToken)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(data))
```
Comment (akomisarek) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java]: We have got almost exact code as above. 

Could we maybe do: content.getBytes(StandardCharsets.UTF_8) 9as the content implementation is doing, and have this further simplified.

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java [hunk 2]
```diff
@@ -90,26 +83,4 @@ void aboutToSubmitShouldPopulateListServedCaseManagementOrdersWhenSendsToAllPart
                     .build())
                 .build()));
     }
-
-    private CallbackRequest buildCallbackRequest(Map<String, Object> data) {
-        return CallbackRequest.builder()
-            .eventId(ACTION_CASE_MANAGEMENT_ORDER.getId())
```
Comment (akomisarek) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java]: how does it work with removal of eventId? by coincidence now? or was the scenario not checked?

Comment (tomaszpowroznik) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java]: good question, checked and it looks that service is ding 
if (DRAFT_CASE_MANAGEMENT_ORDER.getId().equals(eventId)) {
            progressDraftCaseManagementOrder(caseDetails, caseData.getCaseManagementOrder());
        } else {
            progressActionCaseManagementOrder(caseDetails, caseData);
        }
so it works same without eventId,
But yes, it is not right and eventId should be pass by test

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java (hunk 1 above)
Comment (tomaszpowroznik) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java]: done

Comment (tomaszpowroznik) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractControllerTest.java]: done

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!groovy
 
-@Library("Infrastructure")
+@Library("Infrastructure@disableprdepcheck")
```
Comment (akomisarek) [inline Jenkinsfile_CNP]: No longer required.

Comment (tomaszpowroznik) [reply Jenkinsfile_CNP]: done

**Pull request merged**